skeleton/gitops-template/components/http/base/rhoai/notebook.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: kubeflow.org/v1
```

Comment:
quick question: do we need to ensure that this is installed before attempting to create this CR?

Reply:
The help description for the RHOAI checkbox says to ensure that RHOAI is installed before proceeding. I think that's probably sufficient for now (as we also say the same for the GPU operator when installing vllm)